src/server/meme.js

```diff
@@ -18,4 +18,59 @@ module.exports = psql => ({
       return 'unexpected error when trying to save meme data';
     }
   },
+
+  addComment: async (comment, user) => {
+    try {
+      const meme = await psql('memes')
+        .where({ meme_id: comment.meme_id });
+
+      if (meme.length <= 0) {
+        return 'meme with that id does not exist';
+      }
+
+      const newComment = await psql('comments')
+        .insert({
+          user_id: user.user_id,
+          meme_id: comment.meme_id,
+          text: comment.text,
+          date_created: new Date(),
+        }).returning('*');
+
+      return {
+        status: true,
+        comment: newComment,
+      };
+    } catch (e) {
+      console.log(e);
+      return 'unexpected error when trying to add comment';
+    }
+  },
+
+  getMemeComments: async (data) => {
+    try {
+      const comments = await psql('comments')
+        .where({ meme_id: data.meme_id })
+        .orderBy('date_created')
+        .limit(data.limit)
```

Comment:
Do we want the limit to be sent in the request? Shouldn't it be a constant defined in meme.js?

Reply:
Sure thing, I'll make it static